how-pyth-works/oracle-program.md

```diff
@@ -0,0 +1,16 @@
+# Oracle Program
```

Comment:
I added this page mostly because it makes the left nav menu make more sense. At the moment, the nav menu goes from "cross chain" to "Solana Account Structure". I think there needs to be a transition between those.